src/plugins/discover/public/application/angular/doc_table/create_doc_table_react.tsx

```diff
@@ -88,7 +88,6 @@ export function convertDirectiveToRenderFn(
     return () => {
       if (!rejected) {
         // for cleanup
-        // http://roubenmeschian.com/rubo/?p=51
```

Comment:
Can we open these link removal changes as a separate PR?

Reply:
Address this issue with PR: https://github.com/opensearch-project/OpenSearch-Dashboards/pull/3847.  Thanks!

Reply:
Let's rebase this PR after #3847 is merged in.